staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/conversion_test.go

```diff
@@ -526,3 +608,214 @@ func Test_ConvertJSONSchemaPropsToOpenAPIv2Schema(t *testing.T) {
 		})
 	}
 }
+
+// TestKubeOpenapiRejectionFiltering tests that the CRD openapi schema filtering leads to a spec that the
+// kube-openapi/pkg/util/proto model code support in version used in Kubernetes 1.13.
+func TestKubeOpenapiRejectionFiltering(t *testing.T) {
+	for i := 0; i < 10000; i++ {
+		t.Run(fmt.Sprintf("iteration %d", i), func(t *testing.T) {
+			f := fuzz.New()
+			seed := time.Now().UnixNano()
+			//seed = int64(1550678935445547778)
```

Comment:
nit: will clean it up

Reply:
I always left this here to reproduce issues. Feel free to remove it.